### PR TITLE
define welcome page and context root

### PR DESCRIPTION
**Pull request by BertPie**

### CODE REVIEW

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java [hunk 1]
```diff
@@ -105,7 +105,7 @@ private void addReservation(Integer tableId, Integer minutesBeforeNow, Integer d
         Table table = tableDao.findById(tableId);
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java]: MAJOR Remove this useless assignment to local variable "table". [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "table" local variable. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ private void addReservation(Integer tableId, Integer minutesBeforeNow, Integer d
...
         LocalDateTime start = LocalDateTime.now().minusMinutes(minutesBeforeNow);
         LocalDateTime stop = start.plusMinutes(duration);
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java]: MAJOR Remove this useless assignment to local variable "stop". [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1854)
MINOR Remove this unused "stop" local variable. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1481)

File: ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java [hunk 3]
```diff
@@ -105,7 +105,7 @@ private void addReservation(Integer tableId, Integer minutesBeforeNow, Integer d
...
-        reservationDao.save(new Reservation(table, start, stop, ""));
+//        reservationDao.save(new Reservation(table, start, stop, ""));
```
Comment (isacademyci) [inline ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by isacademyci** [comment]
SonarQube analysis reported 10 issues
* ![CRITICAL][CRITICAL] 2 critical
* ![MAJOR][MAJOR] 6 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 5 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [DevPanelServlet.java#L96](https://github.com/infoshareacademy/jjdd5-niewiem/blob/2d1a1e965e31605fdda92bdc026a1a80d37706f6/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L96): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)
1. ![CRITICAL][CRITICAL] [DevPanelServlet.java#L100](https://github.com/infoshareacademy/jjdd5-niewiem/blob/2d1a1e965e31605fdda92bdc026a1a80d37706f6/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L100): Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AS1186)
1. ![MAJOR][MAJOR] [DevPanelServlet.java#L72](https://github.com/infoshareacademy/jjdd5-niewiem/blob/2d1a1e965e31605fdda92bdc026a1a80d37706f6/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L72): Remove this unused private "addOngoingGames" method. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [DevPanelServlet.java#L96](https://github.com/infoshareacademy/jjdd5-niewiem/blob/2d1a1e965e31605fdda92bdc026a1a80d37706f6/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L96): Remove this unused private "addHistory" method. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
1. ![MAJOR][MAJOR] [DevPanelServlet.java#L100](https://github.com/infoshareacademy/jjdd5-niewiem/blob/2d1a1e965e31605fdda92bdc026a1a80d37706f6/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L100): Remove this unused private "addUpcomingReservations" method. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: ClubManagerApp/src/test/java/com/infoshareacademy/niewiem/HallTest.java [hunk 4]
```diff
@@ -1,6 +1,16 @@
 package com.infoshareacademy.niewiem;
 
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
 public class HallTest {
 
+    @Test
+    public void test(){
```
Comment (MichalNowakowski) [inline ClubManagerApp/src/test/java/com/infoshareacademy/niewiem/HallTest.java]: teraz mozemy ten test usunac

**Comment by isacademyci** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [DevPanelServlet.java#L4](https://github.com/infoshareacademy/jjdd5-niewiem/blob/e7d88d9b1281c0c6cc0587a00aa01eba7522a8bd/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L4): Remove this unused import 'com.infoshareacademy.niewiem.dao.ReservationDao'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [DevPanelServlet.java#L9](https://github.com/infoshareacademy/jjdd5-niewiem/blob/e7d88d9b1281c0c6cc0587a00aa01eba7522a8bd/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L9): Remove this unused import 'com.infoshareacademy.niewiem.pojo.Reservation'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [DevPanelServlet.java#L22](https://github.com/infoshareacademy/jjdd5-niewiem/blob/e7d88d9b1281c0c6cc0587a00aa01eba7522a8bd/ClubManagerWeb/src/main/java/com/infoshareacademy/niewiem/web/DevPanelServlet.java#L22): Remove this unused import 'java.time.LocalDateTime'. [rule](http://localhost:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**